typescript/segmented-cfmm/api-extractor.json

```diff
@@ -0,0 +1,364 @@
+/**

```

Comment:
Do we have to include this file in the repo?

Reply:
Probably not, this is from the `api-extract init` command. Removed this for now.